crates/uv-resolver/src/lock/mod.rs

```diff
@@ -1930,6 +1934,12 @@ impl Package {
             table.insert("resolution-markers", value(wheels));
         }
 
+        if let Some(markers) =
+            SimplifiedMarkerTree::new(requires_python, self.markers.clone()).try_to_string()
+        {
+            table.insert("markers", value(markers));
+        }
```

Comment:
Ah interesting. I think I was naively expecting that the "full" markers would just be written in place of the markers we're writing already, instead of adding a new field.

Reply:
Oh I guess I didn’t push the second commit that removed them. One sec.